CMakeLists.txt

```diff
@@ -9,12 +9,28 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
 set(CMAKE_AUTOUIC ON)
 
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake)
 
 find_package(Qt5 COMPONENTS Core Widgets Gui OpenGL REQUIRED)
 find_package(OpenCV REQUIRED)
 find_package(OpenGL REQUIRED)
-find_package(LibAV REQUIRED)
+find_package(Threads REQUIRED)
+
+IF (WIN32)
+    find_package(LibAV REQUIRED)
+ELSE (WIN32)
+    find_path(LIBAV_INCLUDE_DIR libavformat/avformat.h REQUIRED)
```

Comment:
Optional: Alternatively you can modify FindAV to ignore LIBAV_DIR because it doesn't exist on linux.